packages/core/src/translations/client/__mocks__/getTranslations.fixtures.js

```diff
@@ -0,0 +1,29 @@
+import join from 'proper-url-join';
+import moxios from 'moxios';
+
+export default {
+  success: (query, response) => {
+    moxios.stubRequest(
+      join('/api/v1/search/translations', {
+        query,
+      }),
+      {
+        method: 'get',
```

Comment:
'method' doesn't exist inside 'stubRequest' arguments.
To identify the method type of the request, create a key 'get' with success and failure inside.